.changeset/green-rockets-lie.md

```diff
@@ -0,0 +1,16 @@
+---
+"thirdweb": minor
+---
+
+Exposes autoConnect as a standalone function for use outside of react.
+
+```tsx
+import { autoConnect } from "thirdweb/wallets";
+ 
+const walletManager = createConnectionManager();
```

Comment:
do we actually need the connection manager for a non-react environment?

its pretty internal to us, would rather not expose it as a public api if we can. we could do what we've done for the react / react native hooks which 'hide' the connection manager (only used by the core function)